content/cli/smartmodules/hub.md

```diff
@@ -54,16 +54,19 @@ The command **fluvio hub list** will list SmartModules and versions available fo
 
 {{% inline-embed file="embeds/cli/help/fluvio-hub-list.md" %}}
 
-The list command shows Smartmodules by **group/package@version**. Packages uploaded by InfinyOn will be in the
-infinyon group.
+The list command shows Smartmodules by **group/package@version** in the SMARTMODULE column. The Visibility column describes if the package is accessible to
+to the public or private.  If package visibility is **private** only your user login can list or download the package. Packages uploaded by InfinyOn will be in the infinyon group.
```

Comment:
`or private` -> `or only to the group`.

Reply:
right now the hub only supports one group member, the owner